systers_portal/community/forms.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = RequestCommunity
-        fields = ('is_member', 'email_id', 'email', 'name', 'slug', 'order',
```

Comment:
I think making slug editable in the edit form is a better option in case custom slug is required. The issue #307 also suggests adding such behaviour optionally.